WordPressAuthenticator.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressAuthenticator"
-  s.version       = "1.31.0-beta.5"
+  s.version       = "1.31.0-beta.6"
```

Comment:
Since the new release was cut today, this should probably be `1.32.0-beta.1` after @jkmassel finishes the releases.